components/edit-collective/sections/Policies.js

```diff
@@ -14,16 +14,15 @@ import { parseToBoolean } from '../../../lib/utils';
 import Container from '../../Container';
 import { Flex } from '../../Grid';
 import MessageBoxGraphqlError from '../../MessageBoxGraphqlError';
+import RichTextEditor from '../../RichTextEditor';
 import StyledButton from '../../StyledButton';
 import StyledInputField from '../../StyledInputField';
 import StyledSelect from '../../StyledSelect';
-import StyledTextarea from '../../StyledTextarea';
 import { H3, P } from '../../Text';
 
 import { getSettingsQuery } from './EditCollectivePage';
 
-const EXPENSE_POLICY_MAX_LENGTH = 16000; // max in database is ~15,500
-const CONTRIBUTION_POLICY_MAX_LENGTH = 3000; // 600 words * 5 characters average length word
+const MAX_LENGTH = 50000; // max in database is ~15,500 without html
```

Comment:
50000 is the DB limit for the row content. For the characters count itself we should keep the existing ones